bamboo/unit_tests/test_unit_layer_identity_distconv.py

```diff
@@ -85,7 +85,8 @@ def construct_model(lbann):
     x = x_lbann
     x = lbann.Reshape(x, dims="4 4 3")
     y = lbann.Identity(x, data_layout='data_parallel',
-                       parallel_strategy=create_parallel_strategy(4))
+                       parallel_strategy=create_parallel_strategy(
+                           lbann.contrib.lc.systems.gpus_per_node()))
```

Comment:
Similarly for here, I know that we are only testing on LLNL systems, but this should be generalizable.